lib/init.php

```diff
@@ -113,7 +113,7 @@ function gutenberg_site_editor_menu() {
  * @param WP_Admin_Bar $wp_admin_bar Core class used to implement the Toolbar API.
  */
 function modify_admin_bar( $wp_admin_bar ) {
-	if ( gutenberg_use_widgets_block_editor() ) {
+	if ( gutenberg_use_widgets_block_editor() && ! is_admin() ) {
```

Comment:
```suggestion
	if ( gutenberg_use_widgets_block_editor() && $wp_admin_bar->get_node( 'widgets' ) !== null ) {
```

We should directly check if the item we're changing exists, instead of assuming it will always exist on the front-end. Because it doesn't for users that don't have the required capability for editing widgets.

Reply:
Good catch @vdwijngaert 
I just added a commit for that.

Thanks